examples/websockets/src/main.rs

```diff
@@ -63,8 +67,7 @@ async fn main() {
                     )
                 }),
         )
-        // routes are matched from bottom to top, so we have to put `nest` at the
-        // top since it matches all routes
+        // routes are matched from bottom to top
```

Comment:
This is no longer true actually and haven't been for a while. So we can just remove this comment.

```suggestion
```